src/store.js

```diff
@@ -12,6 +12,7 @@ import { NETWORK } from './constants';
 
 
 const ACCESS_DATA_KEY = 'asyncstorage:access';
+const REGISTERED_TOKENS_KEY = 'asyncstorage:registeredTokens';
```

Comment:
We'll need to migrate users when they're updating their apps

Reply:
Done